docs/hardware/pcb_design.md

```diff
@@ -0,0 +1,76 @@
+## **TITLE: PCB BOARD DESIGN**
+### **DESIGN OBJECTIVE**
```

Comment:
Let's not use "title" and "bold" together., also not "all upper case".

Instead of `### **DESIGN OBJECTIVE**` let's just use `### Design objective`

(applies to all headings)